applications/thanos/base/query-deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         - query
         - --grpc-address=0.0.0.0:10901
         - --http-address=0.0.0.0:9090
+        - --endpoint=0.0.0.0:10901
```

Comment:
This needs to be the svc you have made

Reply:
i.e. `prometheus-service.monitoring.svc.cluster.local:10901`

Reply:
gotchaa